tests/test_gondolin.py

```
@@ -0,0 +1,1340 @@
+
+
+class ExecutionResult(TimestampedModel):
+    """Result model for code execution."""
+
+    def __init__(
+        self,
+        stdout: str,
+        stderr: str,
+        exit_code: int,
+        duration_ms: int,
+        status: ExecutionStatus,
```

Ruff (0.15.2)

[error] Redefinition of unused `status` from line 17: `status` redefined here

(F811)